cli/go.mod

```
@@ -16,19 +16,19 @@
 // under the License.
 module github.com/apache/brooklyn-client/cli
 
-go 1.20
+go 1.22.5
```

```suggestion
go 1.22
```

This makes the pipeline to fail with
```
invalid go version '1.22.5': must match format 1.23
```

Only the minor version must be defined